### DISCUSSION TITLE
[QUESTION] Use NFS shared folder mounted to each node for adding new disk in Longhorn

### DISCUSSION THREAD

**Discussion by lookfortonylong** [longhorn/longhorn#5946] [General]
## Question
> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.
Hi everyone,
I have an NFS server and K8s cluster. I can mount the nfs shared folder to each worker nodes. (/data/).
Later I deploy Longhorn, add new disk on nodes by using the mounted folder: /data/
However when I deploy wordpress using helm, the volume created by wordpress and mariadb shows faulted.
Note: it's working if I use local device on each nodes as the volume.
May I know that if I can use mounted folder on each node (/data) as the new disk on Longhorn?
If yes, what may be the issue for faulted message. Thanks.
## Environment

 - Longhorn version: v1.4.1
 - Kubernetes version: v1.24.12+rke2r1
 - Node config
   - OS type and version ubuntu 5.15.0-71-generic
   - CPU per node: 2
   - Memory per node: 6
   - Disk type LVM
   - Network bandwidth and latency between the nodes: LAN
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): VMWare

**Comment by ChanYiLin** [comment]
Hi @lookfortonylong 
No it is not a valid way to use NFS mount point as longhorn disk.

**Reply by lookfortonylong** [reply]
Hi ChanYiLin,
Thanks for your response. I read again the Longhorn document, it seem that NFS is just for backup stuff?
Btw, I have tried to deploy iSCSI server and add the iSCSI disk to the worker nodes, it's working.

**Reply by ChanYiLin** [reply]
Yes, 
you can use NFS or s3 compatible object storage to store the volume backup and restore the volume back.

**Reply by lookfortonylong** [reply]
Thanks ChanYiLin. I'm clear now the way it works.
